plugins/meta/dnsname/files.go

```diff
@@ -145,6 +145,84 @@ func appendToFile(path, podname string, aliases []string, ips []*net.IPNet) erro
 	return nil
 }
 
+func removeHostLinesByIP(path string, ips []*net.IPNet) (bool, error) {
```

Comment:
It is unclear what this function returns. Use named return values to make it clearer.